src/main/java/Duke/ExceptionHandler.java

```diff
@@ -0,0 +1,16 @@
+package Duke;
+
+public class ExceptionHandler {
```

Comment:
Possibly rename? You are not doing any exception handling here. This class simply provides error messages, so you can name it as such